packages/@atjson/renderer-hir/src/index.ts

```diff
@@ -251,6 +241,18 @@ export default class Renderer {
     }
   }
 
+  /**
+   * Get a slice document by the `SliceAnnotation` id.
+   * Useful for grabbing slices like captions / credit / etc
+   * and rendering them in place.
+   *
+   * @param sliceId The id of the slice to return
+   * @returns The slice document or null if there's no slices that match
+   */
+  slice(sliceId: string) {
```

Comment:
would prefer a name like `getSlice` to avoid confusion with Array.prototype.slice